config/prometheus/README.md

```diff
@@ -0,0 +1,15 @@
+# Prometheus Exporter Prerequisites
+
+## Configure Prometheus to scrape the collector
+
+Using the basic configuration provided in [these docs](https://prometheus.io/docs/prometheus/latest/getting_started/#configuring-prometheus-to-monitor-itself) instead of:
+
+```
+static_configs:
+    - targets: ['localhost:<PROMETHEUS_PORT>']
+```
+edit the config target the collector:
+```
+static_configs:
+    - targets: ['localhost:<COLLECTOR_PORT>']
+```
```

Comment:
I do not think the linked doc is very helpful. We are not configuring prometheus to scrape itself.

I think this section can just say that "prometheus should be configured to scrape the exporter's configured ip and port. Its possible that the user's prometheus server is also scraping other targets.